include/dlaf/sender/transform_mpi.h

```diff
@@ -11,6 +11,10 @@
 
 #include <type_traits>
 
+#ifdef EXTRA_MPI_TYPES_DEBUGGING
+#include <pika/debugging/demangle_helper.hpp>
+#endif
+#include <pika/debugging/print.hpp>
```

Comment:
I understand that these can be useful for debugging, but I would prefer if we leave these out at this point, especially since you're planning to change the debugging/logging facilities in pika sometime soon.